drivers/SmartThings/philips-hue/src/init.lua

```diff
@@ -762,24 +789,27 @@ local function do_bridge_network_init(driver, bridge_device, bridge_url, api_key
 
           if connectivity_status.data and #connectivity_status.data > 0 then
             scanned = true
-          end
-
-          for _, status in ipairs(connectivity_status.data) do
-            local hue_device_id = (status.owner and status.owner.rid) or ""
-            log.trace(string.format("Checking connectivity status for device resource id %s", hue_device_id))
-            local child_device = child_device_map[hue_device_id]
-            if child_device then
-              if status.status == "connected" then
-                child_device.log.info_with({hub_logs=true}, "Marking Online after SSE Reconnect")
-                child_device:online()
-                child_device:set_field(Fields.IS_ONLINE, true)
-              elseif status.status == "connectivity_issue" then
-                child_device.log.info_with({hub_logs=true}, "Marking Offline after SSE Reconnect")
-                child_device:set_field(Fields.IS_ONLINE, false)
-                child_device:offline()
+            for _, status in ipairs(connectivity_status.data) do
+              local hue_device_id = (status.owner and status.owner.rid) or ""
+              log.trace(string.format("Checking connectivity status for device resource id %s", hue_device_id))
+              local child_device = child_device_map[hue_device_id]
```

Comment:
I believe this cached value is where we were sending device health changes for deleted devices that @NoahCornell  was seeing, but I'm not fully confident because I would expect to get crashes on indexing in to the device-scoped log first. So I'm not 100% on that yet.